playwright/e2e/one-to-one-chat/one-to-one-chat.spec.ts

```diff
@@ -45,6 +45,7 @@ test.describe("1:1 chat room", () => {
         await expect(
             page.getByRole("group", { name: "Rooms" }).locator(".mx_RoomTile").getByText(user2.displayName),
         ).not.toBeVisible();
+        await page.waitForTimeout(500); // avoid race condition with routing
```

Comment:
what's routing?

Reply:
https://dev.to/marcomonsanto/routing-in-spas-173i

Reply:
yes, but do we have such a thing in Element?